package.json

```diff
@@ -137,13 +138,14 @@
     "tap": "15.0.9",
     "tape": "5.3.1",
     "tape-promise": "4.0.0",
-    "ts-jest": "28.0.2",
+    "ts-jest": "27.0.3",
     "ts-loader": "9.2.5",
     "ts-node": "10.7.0",
     "typescript": "4.3.5",
     "webpack": "5.50.0",
     "webpack-bundle-analyzer": "4.4.2",
-    "webpack-cli": "4.7.2"
+    "webpack-cli": "4.7.2",
+    "yui-lint": "^0.2.0"
```

Comment:
@micoferdinand98 What is the `yui-lint` being used for?

Reply:
that part was automatically included when I installed another license checker package. but since its not relevant I have removed it from the code.